components/common/ConnectWallet/WalletAssetIdentity.vue

```diff
@@ -1,16 +1,32 @@
 <template>
   <div class="flex items-center justify-between">
-    <IdentityItem
-      :account="account"
-      :label="display || shortenedAddress"
-      :prefix="urlPrefix"
-      variant="button" />
-    <div class="flex flex-col">
+    <NuxtLink
+      :to="`/${urlPrefix}/u/${account}`"
+      class="w-full"
+      @click="closeModal">
+      <IdentityItem
+        :account="account"
+        :label="display || shortenedAddress"
+        :prefix="urlPrefix">
+        <template #default="{ label }">
+          <div class="pl-3">
+            <div class="font-bold mb-1">
+              {{ label }}
+            </div>
+
+            <div class="text-xs text-k-grey">
+              {{ $t('viewProfile') }}
```

Comment:
already default value in `IdentityItem`

Reply:
removed the "variant" prop from `IdentityItem` , from what I could see that wasn't used elsewhere.

now is using a slot , passing this "variant", so no default prop, do you think I'd be better to have Inside , like before ?


![CleanShot 2024-01-24 at 13 41 21@2x](https://github.com/kodadot/nft-gallery/assets/44554284/db1e6e10-574d-4e31-90f7-c669439a6288)

Reply:
yes! Imo it's better to use `slot` and refactor component when we have unused stuff